tasks/config_nexus.yml

```diff
@@ -67,6 +68,9 @@
   with_items:
     - "{{ nexus_scheduled_tasks }}"
 
+- name: NEXUS | System -> Http
+  import_tasks: action/system/http.yml
+
```

Comment:
IMO, there should be a variable to toggle this action

Reply:
And what should this variable toggle? The logic is in groovy file and IMHO it's better to run task every time.